vmdb/spec/models/miq_widget_spec.rb

```diff
@@ -132,6 +132,33 @@
         result[@group2].should match_array(users)
       end
 
+      it 'ignores the user that does not exist any more' do
+        ws = FactoryGirl.create(:miq_widget_set, :name => "Home", :userid => @user1.userid, :group_id => @group1.id)
+        @widget_report_vendor_and_guest_os.make_memberof(ws)
```

Comment:
@lfu These two lines are repeated at the top of every test in this context except the first one.  Let's create a sub-context and move this logic into a before block.